<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Spheres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="sphereList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addSphere">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add a new sphere</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/gtango/list-add.svg</normaloff>:/icons/gtango/list-add.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeSphere">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Remove selected spheres...</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/gtango/list-remove.svg</normaloff>:/icons/gtango/list-remove.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="detectHighlights">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Find highlights</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/gtango/applications-system.svg</normaloff>:/icons/gtango/applications-system.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showSpheres">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Load light directions .lp file</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/gtango/spherelight.svg</normaloff>:/icons/gtango/spherelight.svg</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveLP">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Save light directions</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/gtango/document-save.svg</normaloff>:/icons/gtango/document-save.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="imageList1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="imageList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>4</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionDetectHighlights"/>
    <addaction name="actionLoad_LP"/>
    <addaction name="actionSave_LP"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="actionToggle_max_luma"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionView_RTI"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_sphere_point"/>
    <addaction name="actionAdd_sphere_highlight"/>
    <addaction name="actionDetectHighlights"/>
    <addaction name="separator"/>
    <addaction name="actionRuler"/>
    <addaction name="actionDelete_selected"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_all_right"/>
    <addaction name="actionRotate_all_right_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCalibration">
    <property name="title">
     <string>Calibration</string>
    </property>
    <addaction name="actionLens_parameters"/>
    <addaction name="actionWhite_balance"/>
    <addaction name="actionDome_geometry"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExport_RTI"/>
    <addaction name="actionExport_Normals"/>
    <addaction name="separator"/>
    <addaction name="actionShow_queue"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit_2"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCalibration"/>
   <addaction name="menuExport"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="separator"/>
   <addaction name="actionRuler"/>
   <addaction name="actionExport_RTI"/>
   <addaction name="actionShow_queue"/>
   <addaction name="actionView_RTI"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/folder.svg</normaloff>:/icons/feather/folder.svg</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/chevron-right.svg</normaloff>:/icons/feather/chevron-right.svg</iconset>
   </property>
   <property name="text">
    <string>Next image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/chevron-left.svg</normaloff>:/icons/feather/chevron-left.svg</iconset>
   </property>
   <property name="text">
    <string>Previous image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/zoom-in.svg</normaloff>:/icons/feather/zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/zoom-out.svg</normaloff>:/icons/feather/zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionAdd_sphere_point">
   <property name="text">
    <string>Add sphere point</string>
   </property>
  </action>
  <action name="actionAdd_sphere_highlight">
   <property name="text">
    <string>Add sphere highlight</string>
   </property>
  </action>
  <action name="actionDelete_selected">
   <property name="text">
    <string>Delete selected item</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/info.svg</normaloff>:/icons/feather/info.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDetectHighlights">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/settings.svg</normaloff>:/icons/feather/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Find highlights</string>
   </property>
  </action>
  <action name="actionSave_LP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/save.svg</normaloff>:/icons/feather/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save LPs...</string>
   </property>
  </action>
  <action name="actionExport_RTI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/download.svg</normaloff>:/icons/feather/download.svg</iconset>
   </property>
   <property name="text">
    <string>Export RTI...</string>
   </property>
  </action>
  <action name="actionLoad_LP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/folder.svg</normaloff>:/icons/feather/folder.svg</iconset>
   </property>
   <property name="text">
    <string>Load LP...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/save.svg</normaloff>:/icons/feather/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="toolTip">
    <string>Save project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/file.svg</normaloff>:/icons/feather/file.svg</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>New project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRuler">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/gtango/ruler.svg</normaloff>:/icons/gtango/ruler.svg</iconset>
   </property>
   <property name="text">
    <string>Measure...</string>
   </property>
   <property name="toolTip">
    <string>Take a measure</string>
   </property>
  </action>
  <action name="actionToggle_max_luma">
   <property name="text">
    <string>Toggle max luma</string>
   </property>
  </action>
  <action name="actionLens_parameters">
   <property name="text">
    <string>Lens parameters...</string>
   </property>
  </action>
  <action name="actionRotate_all_right">
   <property name="text">
    <string>Rotate right all images</string>
   </property>
  </action>
  <action name="actionRotate_all_right_2">
   <property name="text">
    <string>Rotate left all images</string>
   </property>
  </action>
  <action name="actionView_RTI">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/cast.svg</normaloff>:/icons/feather/cast.svg</iconset>
   </property>
   <property name="text">
    <string>View RTI </string>
   </property>
   <property name="toolTip">
    <string>View RTI in the browser</string>
   </property>
  </action>
  <action name="actionNormals">
   <property name="text">
    <string>Normals</string>
   </property>
  </action>
  <action name="actionExport_RTI_2">
   <property name="text">
    <string>Export RTI...</string>
   </property>
  </action>
  <action name="actionExport_Normals">
   <property name="text">
    <string>Export Normals...</string>
   </property>
  </action>
  <action name="actionShow_queue">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/list.svg</normaloff>:/icons/feather/list.svg</iconset>
   </property>
   <property name="text">
    <string>Show queue...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/feather/settings.svg</normaloff>:/icons/feather/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionWhite_balance">
   <property name="text">
    <string>White balance...</string>
   </property>
  </action>
  <action name="actionDome_geometry">
   <property name="text">
    <string>Dome geometry...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
